rocAL/rocAL/include/pipeline/image.h

```diff
@@ -112,6 +112,9 @@ struct ImageInfo
     uint32_t * get_roi_height() const;
     const std::vector<uint32_t>& get_roi_width_vec() const;
     const std::vector<uint32_t>& get_roi_height_vec() const;
+    const std::vector<uint32_t>& get_original_roi_width_vec() const;
```

Comment:
please change the names to get_original_width_vec() && ..
roi is confusing

Reply:
Changed the function name in the latest commit